connector/src/main/resources/schema/wildfly-jca_4_0.xsd

```diff
@@ -101,7 +101,14 @@
               </xs:annotation>
             </xs:element>
 
-            <xs:element name="cached-connection-manager" type="cached-connection-managerType" minOccurs="0" maxOccurs="1"></xs:element>
+            <xs:element name="cached-connection-manager" type="cached-connection-managerType" minOccurs="0" maxOccurs="1">
+                <xs:annotation>
+                    <xs:documentation>
+                        Cached connection manager settings for resource adapters. NOTE: This service is always available so there's no
+                        way to remove it.
```

Comment:
Because you can actually configure it. My suggestion would be to change `minOccurs` to `1`, but since that requires a schema change I don't know if this issue is the best one to do this.